### PR TITLE
[6.3] FIX RHAI UI tests

### PR DESCRIPTION
**Pull request by ldjebran**
```console
pytest -v tests/foreman/rhai/test_rhai.py 
================================================ test session starts =================================================
platform linux2 -- Python 2.7.14, pytest-3.2.3, py-1.5.2, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/2.7.14/envs/sat-6.3/bin/python2.7
cachedir: .cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.20.1, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1, fauxfactory-1.1.1
collected 3 items                                                                                                     
2017-12-28 18:35:01 - conftest - DEBUG - Collected 3 test cases


tests/foreman/rhai/test_rhai.py::RHAITestCase::test_negative_org_not_selected PASSED
tests/foreman/rhai/test_rhai.py::RHAITestCase::test_positive_register_client_to_rhai <- robottelo/decorators/__init__.py PASSED
tests/foreman/rhai/test_rhai.py::RHAITestCase::test_positive_unregister_client_from_rhai <- robottelo/decorators/__init__.py PASSED

================================================= 0 tests deselected =================================================
============================================ 3 passed in 1476.12 seconds =============================================
```

### CODE REVIEW

File: robottelo/ui/rhai.py [hunk 1]
```diff
@@ -16,3 +17,19 @@ def view_registered_systems(self):
             locators['insights.registered_systems']
         ).text
         return result
+
+    def unregister_system_from_inventory(self, system_name, confirm=True):
+        """Unregister system from inventory"""
+        Navigator(self.browser).go_to_insights_inventory()
```
Comment (oshtaier) [inline robottelo/ui/rhai.py]: any reason to not use our default approach with overloaded method?
e.g.
```
def navigate_to_entity(self):
        """Navigate to Compute Profile entity page"""
        Navigator(self.browser).go_to_compute_profiles()
```

Comment (ldjebran) [reply robottelo/ui/rhai.py]: This is not a pure entity, this a class for many different pages

Comment (oshtaier) [reply robottelo/ui/rhai.py]: maybe we create class per each entity
like:
module - rhai
but
classes - inventory and etc

File: robottelo/ui/rhai.py [hunk 2]
```diff
@@ -16,3 +17,19 @@ def view_registered_systems(self):
...
+        # select the system
+        self.assign_value(
+            locators["insight.inventory.system_checkbox"] % system_name,
+            True
+        )
+        # click on drop-down actions button
+        self.click(locators["insight.inventory.actions_button"])
+        # click on unregister action
+        self.click(locators["insight.inventory.action_unregister"])
+        if confirm:
```
Comment (abalakh) [inline robottelo/ui/rhai.py]: And if `confirm` is False? Is there any modal window which we'll get stuck on? I think you should add some click on `cancel` button if confirm == False.

File: robottelo/ui/rhai.py [hunk 3]
```diff
@@ -16,3 +17,19 @@ def view_registered_systems(self):
...
+            # click on yes dialog button
```
Comment (abalakh) [inline robottelo/ui/rhai.py]: Never thought i'm gonna say this, but i think these extra comments are redundant - commands in this method are very simple and self-descriptive.

File: tests/foreman/rhai/test_rhai.py [hunk 4]
```diff
@@ -138,24 +136,7 @@ def test_positive_unregister_client_from_rhai(self):
...
-                        ajax_timeout=40,
-                    )
-
-                    # Confirm selection for clicking on 'Yes' to unregister the
-                    # system
-                    session.nav.click(
-                        locators['insights.unregister_button']
-                    )
-                    self.browser.refresh()
-                    time.sleep(60)
```
Comment (abalakh) [inline tests/foreman/rhai/test_rhai.py]: So there was a necessity to wait for a minute for action to finish. Did something change in application? Are you sure vm will be updated immediately after ui action with no delays?

Comment (ldjebran) [reply tests/foreman/rhai/test_rhai.py]: I did not saw any delay in that

Comment (ldjebran) [reply tests/foreman/rhai/test_rhai.py]: The tests were very stable

File: robottelo/ui/rhai.py (hunk 3 above)
Comment (ldjebran) [reply robottelo/ui/rhai.py]: that comments are not for me :)

**Comment by ldjebran** [comment]
@oshtaier @abalakh done all the changes
```
pytest -v tests/foreman/rhai/test_rhai.py 
================================================ test session starts =================================================
platform linux2 -- Python 2.7.14, pytest-3.2.3, py-1.5.2, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/2.7.14/envs/sat-6.3/bin/python2.7
cachedir: .cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.20.1, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1, fauxfactory-1.1.1
collected 3 items                                                                                                     
2017-12-29 14:50:25 - conftest - DEBUG - Collected 3 test cases


tests/foreman/rhai/test_rhai.py::RHAITestCase::test_negative_org_not_selected PASSED
tests/foreman/rhai/test_rhai.py::RHAITestCase::test_positive_register_client_to_rhai <- robottelo/decorators/__init__.py PASSED
tests/foreman/rhai/test_rhai.py::RHAITestCase::test_positive_unregister_client_from_rhai <- robottelo/decorators/__init__.py PASSED

================================================= 0 tests deselected =================================================
============================================= 3 passed in 990.15 seconds =============================================
```

**Comment by ldjebran** [comment]
@abalakh removed that confirm arg
```
pytest tests/foreman/rhai/test_rhai.py -v -k "test_positive_unregister_client_from_rhai"
================================================ test session starts =================================================
platform linux2 -- Python 2.7.14, pytest-3.2.3, py-1.5.2, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/2.7.14/envs/sat-6.3/bin/python2.7
cachedir: .cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.20.1, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1, fauxfactory-1.1.1
collected 3 items                                                                                                     
2017-12-29 15:33:11 - conftest - DEBUG - Collected 3 test cases


tests/foreman/rhai/test_rhai.py::RHAITestCase::test_positive_unregister_client_from_rhai <- robottelo/decorators/__init__.py PASSED

================================================= 2 tests deselected =================================================
====================================== 1 passed, 2 deselected in 441.14 seconds ======================================
```

**Pull request merged**